opentdf-ers-mode.yaml

```diff
@@ -0,0 +1,90 @@
+mode: entityresolution
```

Comment:
Maybe add comments here and atop other `-mode.yaml` files about what they are showing off and how to use them, or maybe a pithy comment about why it exists